src/mscorlib/src/System/Threading/ManualResetEventSlim.cs

```diff
@@ -242,7 +241,6 @@ private void Initialize(bool initialState, int spinCount)
         /// </summary>
         private void EnsureLockObjectCreated()
         {
-            Contract.Ensures(m_lock != null);
 
```

Comment:
Nit: it'd be nice if we didn't have blank space like this at the beginning of methods.

Reply:
All regexed away..